<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wittern</class>
 <widget class="QMainWindow" name="Wittern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wittern</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Dex">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Dex</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Scanning...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlexUpdate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Request Flex Update</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="localAdapterBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Local Bluetooth Adapter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="firstAdapter">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="secondAdapter">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>firstAdapter</zorder>
    <zorder>secondAdapter</zorder>
    <zorder>titleLabel</zorder>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>87</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="sendText"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="chat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>211</width>
      <height>311</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="remoteDevices">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UnlockButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Vend Unlock</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-240</x>
      <y>-50</y>
      <width>921</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="n2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="n1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="n3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="n4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="n5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="n6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="n7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ResourceFiles/backgroundjpg.qrc">:/new/bluBack/SubMenuBackground.png</pixmap>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>n7</zorder>
   <zorder>n6</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>chat</zorder>
   <zorder>Cancel</zorder>
   <zorder>status</zorder>
   <zorder>localAdapterBox</zorder>
   <zorder>n2</zorder>
   <zorder>n1</zorder>
   <zorder>connectButton</zorder>
   <zorder>Dex</zorder>
   <zorder>n3</zorder>
   <zorder>n4</zorder>
   <zorder>n5</zorder>
   <zorder>UnlockButton</zorder>
   <zorder>FlexUpdate</zorder>
   <zorder>quitButton</zorder>
   <zorder>remoteDevices</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWittern_Market">
    <property name="title">
     <string>Wittern Market</string>
    </property>
   </widget>
   <addaction name="menuWittern_Market"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ResourceFiles/backgroundjpg.qrc"/>
 </resources>
 <connections/>
</ui>
